projects/fw_102/src/main.c

```diff
@@ -0,0 +1,38 @@
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define COUNTER_DURATION 500
+
+typedef struct {
+  uint8_t counter_a;
+  uint8_t counter_b;
+  int turn;
+} Counters;
+
+void prv_timer_callback(SoftTimerId timer_id, void *context) {
```

Comment:
also, if a function is only going to be used in the file it is declared, you should declare it as `static void`. You've got the `prv_` naming convention correct though 🙂